nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -371,4 +399,78 @@ private class LogEntryEventHandler_Tests {
             System.assert(topicsSet.contains(topicAssignment.Topic.Name));
         }
     }
+
+    @isTest
+    static void it_should_set_api_release_number_and_api_release_version_from_callout() {
+        Test.setMock(HttpCalloutMock.class, new SuccessCalloutMock());
+        LogEntryEventHandler.shouldCallStatusApi = true;
+
+        String transactionId = '123-456-789-0';
+        LogEntryEvent__e logEntryEvent = new LogEntryEvent__e(Message__c = 'my message', Timestamp__c = System.now(), TransactionId__c = transactionId);
+
+
+        Database.SaveResult saveResult;
+
+        Test.startTest();
+
+        saveResult = EventBus.publish(logEntryEvent);
+        // Normally, you don't have to call Test.getEventBus().deliver() if you're also using Test.stopTest()
+        // But, in this case, there are 3 transactions happening (original test, async platform event, and async future method)
```

Comment:
thank you for including this note - super helpful

Reply:
Yeah, I wasted several hours on this trying to get tests to pass - but as soon as I added the calls for `Test.getEventBus().deliver()`, everything finally worked. I thought an extra note would be useful in avoiding this headache again.